bitcoind-rpc/src/main/scala/org/bitcoins/rpc/config/BitcoindConfig.scala

```diff
@@ -23,8 +24,25 @@ import java.nio.file.Path
   *
   * @see https://github.com/bitcoin/bitcoin/blob/master/doc/bitcoin-conf.md
   */
-abstract class BitcoindConfig extends BitcoinSLogger {
-  private[bitcoins] def lines: Seq[String]
+case class BitcoindConfig(lines: Seq[String], datadir: File)
```

Comment:
Might make sense to make `lines` `private[bitcoins]`, as that's not really a field it makes sense to expose to end users